src/resmom/linux/alps.c

```diff
@@ -5815,9 +5815,9 @@ alps_confirm_reservation(job *pjob)
 int
 alps_cancel_reservation(job *pjob)
 {
-	char buf[1024];
-	basil_response_t *brp;
-	char	filename[MAXPATHLEN];
+	char			buf[1024];
+	basil_response_t	*brp;
+	char			filename[MAXPATHLEN];
```

Comment:
the above still doesn't look aligned to me. Is it github or they are actually not aligned?

Reply:
It's aligned. I think it's just github.

Reply:
Ok.